visual-testing-app/src/components/page-content-container-narrow/page-content-container-narrow.visualroute.tsx

```diff
@@ -0,0 +1,33 @@
+import { PageContentNarrow } from '@commercetools-frontend/application-components';
+import { ThemeProvider } from '@commercetools-uikit/design-system';
+import { Suite, Spec } from '../../test-utils';
+
+export const routePath = '/page-content-container-narrow';
+
+const Box = () => {
+  return (
+    <div
+      style={{
+        width: '100%',
+        height: '75vh',
+        backgroundColor: 'coral',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+    >
+      <h2>Page content container narrow</h2>
+    </div>
+  );
+};
+
+export const Component = () => (
+  <Suite>
+    <ThemeProvider theme="test" />
```

Comment:
I think for VRTs it makes sense to check the new theme, since the default theme just renders a `<div>`.
After the migration, we can remove the `ThemeProvider` from here.